src/UI/Card.jsx

```diff
@@ -1,5 +1,15 @@
-const Card = () => {
-  return <></>
+import style from './Card.module.scss'
+
+const Card = ({title, description, icon, modifier}) => {
+  return <div className={`${style.card} ${style[`card--${modifier}`]}`}>
+    <div className={style.summary}>
+      <h3>{title}</h3>
+      <p>{description}</p>
+    </div>
+    <div className={style.image}>
+      {icon && <img src={icon} alt="A magnifying glass with an eye inside"/>}
```

Comment:
I think the title plus description already describe this card. The "alt" attribute value could be empty here, as it isn't giving any kind of useful information.